### ISSUE TITLE
Add a test suite

### ISSUE THREAD

**Issue by waldyrious** [hamstar/Wikimate#159]
Setting up unit tests, and perhaps also some sort of integration test with real calls to the MediaWiki API, would be welcome in providing confidence about the correct functioning of Wikimate, as well as prevent regressions.

This would also allow us to explicitly test Wikimate against different versions of PHP, which would help with #158.

**Comment by waldyrious** [comment]
Related: #137.

**Comment by Xymph** [comment]
I agree, but this requires another developer, as it's something I have little knowledge about, and no time for.
